github_api_helper.js

```diff
@@ -46,13 +46,21 @@ function getPRFiles (context, number) {
 /**
  * Get file contents as raw data
  * @param {import('probot').Context} context Probot context
+ *  @param {any[]} pullRequests object which contains information about the pull request
  * @param {string} path file path relative to repository root
  * @param {string} ref sha of file revision
  * @returns {Promise<any>} GitHub response
  * See https://developer.github.com/v3/repos/contents/#get-contents
  */
-function getRawFileContents (context, path, ref) {
-  const { owner, repo } = context.repo()
+function getRawFileContents (context, pullRequest, path, ref) {
+  let { owner, repo } = context.repo()
+  // This check is necessary in the case when there is a pr
+  // which is not from forked repository.
+  // Then the repo and owner fields are not in the pullRequest object
```

Comment:
Do not explain this inside the function, add it to the parameter JSdoc
ie `@param {pull_request.head?} head Reference to the fork the commit originated from`

Reply:
I still think this comment is useful but still I will add head as a parameter.